lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -193,20 +239,28 @@ class FatMemManager : public OpSemMemManager {
   /// \return symbolic value of the read integer
   Expr loadIntFromMem(FatPtrTy ptr, FatMemValTy mem, unsigned byteSz,
                       uint64_t align) override {
-    return m_mem.loadIntFromMem(mkRawPtr(ptr), mkRawMem(mem), byteSz, align);
+    return m_main.loadIntFromMem(mkRawPtr(ptr), mkRawMem(mem), byteSz, align);
   }
 
   /// \brief Loads a pointer stored in memory
   /// \sa loadIntFromMem
   FatPtrTy loadPtrFromMem(FatPtrTy ptr, FatMemValTy mem, unsigned byteSz,
                           uint64_t align) override {
-    return mkFatPtr(
-        m_mem.loadPtrFromMem(mkRawPtr(ptr), mkRawMem(mem), byteSz, align));
+    MemValTy rawVal =
+        m_main.loadPtrFromMem(mkRawPtr(ptr), mkRawMem(mem), byteSz, align);
+    MemValTy slot0Val =
+        m_slot0.loadIntFromMem(mkRawPtr(ptr), mkSlot0Mem(mem), byteSz, align);
+    MemValTy slot1Val =
+        m_slot1.loadIntFromMem(mkRawPtr(ptr), mkSlot1Mem(mem), byteSz, align);
+    LOG("opsem", WARN << "ptr:" << rawVal << " slot0:" << slot0Val
```

Comment:
`WARN` is reserved for warnings. Please do not use it for debug. Use `llvm::errs()` for debug instead.

Reply:
ack